### PR TITLE
Double click issue fixed

### LINKED ISSUE #1453: Double click issues
Double click on component while pasting it. The component shows redundant buttons.
Double click on a CO while pasting it. The component shows redundant COs
Double click while logging in. The dashboard shows redundant views.

### PR DESCRIPTION
**Pull request by aniketdharia**
fixed #1453

### CODE REVIEW

**Comment by taylortom** [comment]
@aniketdharia in future, would it be possible to separate any commits which modify whitespace from the actual fix? Makes it a lot easier when trying to review what's been changed :)

File: frontend/src/core/user/models/sessionModel.js [hunk 1]
```diff
@@ -67,10 +67,12 @@ define(function(require) {
 
           Origin.trigger('login:failed', errorCode);
         }
+      }).always(function(){
+        self.set('loginFired',false);
```
Comment (taylortom) [inline frontend/src/core/user/models/sessionModel.js]: Is this code relevant to the fix?

**Comment by taylortom** [comment]
@aniketdharia I've been informed that you can append `?w=1` to the URL to hide whitespace changes, so don't worry too much about my request :)

File: frontend/src/core/user/views/loginView.js [hunk 2]
```diff
@@ -43,6 +43,9 @@ define(function(require) {
 
     submitLoginDetails: function(e) {
       e && e.preventDefault();
+      //To avoid multiple login request.
+      var isLoginInprocess = this.model.get('loginFired');
```
Comment (brian-learningpool) [inline frontend/src/core/user/views/loginView.js]: I'm not sure this should be held as an attribute of the session.

File: frontend/src/core/user/models/sessionModel.js (hunk 1 above)
Comment (brian-learningpool) [inline frontend/src/core/user/models/sessionModel.js]: @aniketdharia, I think it would be better to have a more generic approach for this, whether this involves implementing double-click handlers in the views or something else.

Comment (ambadasavhad) [inline frontend/src/core/user/models/sessionModel.js]: yes ,this is relevant to fix this issue as it helps to set flag on ajax request completion.

**Comment by taylortom** [comment]
@aniketdharia, @ambadasavhad there hasn't been any movement on this for a while, is it still valid?

**Comment by aniketdharia** [comment]
@taylortom will check and get back.
